### PR TITLE
remove `strata` argument and associated slides

### LINKED ISSUE #318: remove `strata` argument and associated slides

### PR DESCRIPTION
**Pull request by simonpcouch**
Closes #318.

### CODE REVIEW

File: slides/intro-04-evaluating-models.qmd [hunk 1]
```diff
@@ -424,20 +424,16 @@ vfold_cv(forested_train, v = 5)
...
 
 ```{r vfold-cv-strata}
-vfold_cv(forested_train, strata = forested)
+vfold_cv(forested_train)
 ```
 
-. . .
-
-Stratification often helps, with very little downside
-
```
Comment (hfrick) [inline slides/intro-04-evaluating-models.qmd:433]: Let's remove this slide entirely since the only point was the stratification

**Review by hfrick — COMMENTED** [review]
Could you also remove it from the associated classwork files, please?

**Comment by simonpcouch** [comment]
Ahhh I had changed my search directory to `slides/` and didn't change it back to the project directory until just now! Woopsies.

**Review by hfrick — APPROVED** [review]
Ah yes, I've also had to do that for a different PR. The archive directory has many many hits 😆

**Pull request merged**